### PR TITLE
Output skipped tests from qunit to xunit

### PR DESCRIPTION
**Pull request by BryanCrotaz**
Outputs skipped tests with <skipped/> tag

### CODE REVIEW

**Comment by johanneswuerbach** [comment]
Thanks :-) Looks like the test failure is related to your changes, could you take a look?

File: lib/ci/test_reporters/xunit_reporter.js [hunk 1]
```diff
@@ -20,7 +21,14 @@ XUnitReporter.prototype = {
       result: data
     })
     this.total++
-    if (data.passed) this.pass++
+    if (data.skipped) 
+    {
```
Comment (johanneswuerbach) [inline lib/ci/test_reporters/xunit_reporter.js]: Could you move the opening `{` to into the same line with the if?

File: lib/ci/test_reporters/xunit_reporter.js [hunk 2]
```diff
@@ -20,7 +21,14 @@ XUnitReporter.prototype = {
       result: data
     })
     this.total++
-    if (data.passed) this.pass++
+    if (data.skipped) 
+    {  
+      this.skipped++
+    }
+    else
+    {
```
Comment (johanneswuerbach) [inline lib/ci/test_reporters/xunit_reporter.js]: Same ^^^

**Comment by BryanCrotaz** [comment]
I'm getting several failed tests on master without my changes but no test
failures when I apply my changes to the 8.x branch

Bryan Crotaz
Silver Curve

On 19 Jul 2015, at 21:15, "Johannes Würbach" <notifications@github.com>
wrote:

Thanks :-) Looks like the test failure is related to your changes, could
you take a look?

—
Reply to this email directly or view it on GitHub
<https://github.com/airportyh/testem/pull/606#issuecomment-122700427>.

**Comment by BryanCrotaz** [comment]
Interesting - on my local windows machine I get two tests failing, but Travis gets none

**Comment by johanneswuerbach** [comment]
Thanks, could you squash your commits?

**Comment by johanneswuerbach** [comment]
Any chance rebasing those commits?

**Pull request closed without merging**